### PR TITLE
Accessibility font

### PR DESCRIPTION
**Pull request by TB516**
Added accessibility font and accessibility options with font toggle to the settings. Additionally added font files and CSS font families to allow changing fonts easily in the future.

### CODE REVIEW

File: client/src/menu/Anchor.tsx [hunk 1]
```diff
@@ -104,13 +105,21 @@ export default function Anchor(props: Readonly<{
     const setFontSize = (n: number) => {
         document.documentElement.style.fontSize = `${n}em`;
     }
+    const setAccessibilityFontEnabled = (enabled: boolean) => {
+        const font = enabled ? 'game-accessible' : 'game-base';
+        const iconFactor = enabled ? '1.2' : '1';
+
+        document.documentElement.style.setProperty('--game-font', font);
+        document.documentElement.style.setProperty('--icon-factor', iconFactor);
+    }
```
Comment (Jack-Papel) [inline client/src/menu/Anchor.tsx:114]: You should also override the kira font

Comment (TB516) [reply client/src/menu/Anchor.tsx:114]: How would I go about doing this? Is there a kira CSS file that has that stuff?

File: client/src/index.css [hunk 2]
```diff
@@ -34,13 +85,13 @@ s {
...
     padding: 0;
-    /* font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif; */
-    font-family: 'Roboto', sans-serif;
     
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
 
     color: white;
+
+    font-family: var(--game-font);
```
[the remark this answers is not in the document]
Comment (TB516) [reply client/src/index.css]: I used the default Arial font as it is a more widely available font, so this should be resolved now.

File: client/src/index.css [hunk 3]
```diff
@@ -34,13 +89,13 @@ s {
...
     padding: 0;
-    /* font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif; */
-    font-family: 'Roboto', sans-serif;
     
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
 
     color: white;
+
+    font-family: var(--game-font, Ariel);
```
Comment (Jack-Papel) [inline client/src/index.css:98]: *Arial

**Review by Jack-Papel — CHANGES_REQUESTED** [review]

File: client/src/menu/Anchor.tsx (hunk 1 above)
Comment (TB516) [reply client/src/menu/Anchor.tsx]: This should be resolved now!

**Comment by TB516** [comment]
Just fixed the spelling of Aria, everything should ba addressed now.

**Review by ItsSammyM — APPROVED** [review]
I leaving this as approved but I request 1 more change

The checkbox and the word font. In the settings menu, they aren't vertically aligned.
Please try to fix this for at least 10 minutes. If you cant figure it out then just merge this because its not a big deal.

Also is it just me or is this font more difficult to read?

**Review by Jack-Papel — APPROVED** [review]

**Pull request merged**